### PR TITLE
Run3-trk09A Modify run21 version tracker flat minus05percent material files by taking care of multiple occurrence of materials in the definition of composite materials

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Modify run21 version tracker flat minus05percent material files by taking care of multiple occurrence of materials in the definition of composite materials

#### PR validation:

Use the cfg's in Geometry/TrackerCommonData/test/python

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Not sure if this needs to be back ported to 12_0_X

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

File: Geometry/TrackerCommonData/test/python/run21flat.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+            'pythia8CommonSettings', 
+            'pythia8CP5Settings', 
+            'processParameters'
+        ),
+        processParameters = cms.vstring(
+            'Top:gg2ttbar = on ', 
+            'Top:qqbar2ttbar = on ', 
+            '6:m0 = 175 '
+        ),
+        pythia8CP5Settings = cms.vstring(
```
Comment (qliphy) [inline Geometry/TrackerCommonData/test/python/run21flat.py:129]: @bsunanda   For "pythia8CP5settings" and "pythia8CommonSettings" below, you may simply load the cfg in refer to e.g.
https://github.com/cms-sw/cmssw/blob/master/Configuration/Generator/python/TTbar_14TeV_TuneCP5_cfi.py#L20-L21

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**